addons/website/static/src/snippets/s_searchbar/search_bar.js

```diff
@@ -229,6 +237,31 @@ export class SearchBar extends Interaction {
             ev.preventDefault();
         }
     }
+
+    /**
+     * Lazy Loading
+     * @params {number} offset
+     * @params {string} search_type
+     * @params {HTMLElement} currentTarget
+     */
+    async loadMore(offset, search_type, currentTarget) {
+        const res = await this.fetch(offset, search_type);
+        if (res) {
+            const bucket = Object.values(res.results)[0];
+            const template = bucket.templateKey;
+            if (bucket.searchCount <= offset) {
```

Comment:
```suggestion
            if (bucket.data.length <= offset) {
```
May be you want to check this condition or may be want to compare with the next offset? ( If not, it is not working when there is multiple sections )